packages/dm-core-plugins/src/list/ListPlugin.tsx

```diff
@@ -59,7 +59,7 @@ const defaultConfig: TListConfig = {
   selectFromScope: undefined,
   hideInvalidTypes: false,
   compact: false,
-  defaultPaginationRowsPerPage: 10,
+  defaultPaginationRowsPerPage: 5,
```

Comment:
Was it requested that we universally change this to a smaller amount? 🙂 Just checking cause this will change the default on anything that uses List